src/test/java/org/snomed/snowstorm/rest/ConceptControllerTest.java

```diff
@@ -662,7 +662,6 @@ public void findConceptHistory_ShouldReturnExpectedConceptHistory_WhenShowFuture
 		});
 		ConceptHistory conceptHistory = responseEntity.getBody();
 		List<ConceptHistory.ConceptHistoryItem> history = conceptHistory.getHistory();
-		ConceptHistory.ConceptHistoryItem januaryRelease = conceptHistory.getConceptHistoryItem("20200131").get();
```

Comment:
This line was unused.

Reply:
On line 498 the componentType could be `Concept` every time.

Reply:
@kaicode , I'm sorry I don't understand. January has 4 component types (Concept, Description, Relationship & Axiom) which is asserted in the for-loop and so it's not always Concept?

Reply:
The key word is _could_. If they were all Concept, or two Description and two Relationship the assertion would still pass right?

Reply:
Thanks, I get you now; I have fixed this by being more explicit in my tests. I encountered a minor gotcha in that my response was being deserialised into a HashSet rather than my defined TreeSet which was of course useless for my tests/assertions. To get around this I added the `@JsonDeserialize` annotation to `ConceptHistory`.